src/CMakeLists.txt

```diff
@@ -340,12 +340,6 @@ find_package(Qt5 COMPONENTS Gamepad QUIET)
 
 if (Qt5Core_VERSION VERSION_LESS 5.7)
   message(FATAL_ERROR "Mudlet requires Qt 5.7 or later")
-else()
-  if (Qt5Core_VERSION VERSION_LESS 5.7)
-    message(WARNING "Qt version: ${Qt5Core_VERSION} - no Gamepad support, it requires 5.7 at a minimum")
-  else()
-      message(STATUS "Qt version: ${Qt5Core_VERSION}")
```

Comment:
Ah, please leave this (and thus the line above) in as an `else` for the outer (remaining) `if` to report what is being used, whilst it is not *too* hard to guess from the CI process when everything is working - it can be helpful when debugging things when AppVeyor/Travis change/break things or an end-user is moaning about something breaking when they try to roll their own code (and does helps me to keep track of which Qt version I have got currently selected).

Reply:
:sob: I have failed to get what I wanted across, which was the whole block of code to be like the following (so, like the original code, if it *was* above the minimum it would report the **actual value used**):
```
if (Qt5Core_VERSION VERSION_LESS 5.7)
    message(FATAL_ERROR "Mudlet requires Qt 5.7 or later")
else()
    message(STATUS "Qt version: ${Qt5Core_VERSION}")
endif()